coverage_assessment.R

```diff
@@ -41,7 +41,118 @@ nut <- get_data()
 
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
+library(dplyr)
+
+## Load functions in R directory ----
+for (f in list.files("R", full.names = TRUE)) source (f)
+
+
+# Read data ----
+nut <- get_data()
+
+
+####MAM
+
+with(nut, sum(muac >= 11.5 & muac <12.5  & age >=6 & age <=59 & oedema == 2, na.rm= TRUE))
```

Comment:
In terms of the age cut-offs for the sample, a more consistent approach is to use `age >= 6 & age < 60`. Your current code for age cut-offs is still correct mainly because the age is in whole months. But not all nutrition data would give age in whole months. If you use `age >= 6 & age <= 59` and the age is not whole months, you may miss some children who are not yet 5 years old (60 months) but are already beyond 59 months (4 years and 11 months and some days old). Whilst this maybe little, it will still be biased as you are missing out on eligible children. This is not relevant to our data because the age is in whole months but it is still good practice to code it as `age >= 6 & age < 60` so that it is universally applicable to any dataset.

Reply:
Agreed. Made changes!